src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.db;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FruitStorage {
+    private static final Map<String, Integer> fruitRepository = new HashMap<>();
+
+    public static Map<String, Integer> getFruits() {
+        return fruitRepository;

```

Comment:
Returning the original map from getFruits() allows external modification of the map. Consider returning a read-only view of the map using Collections.unmodifiableMap(fruitRepository) to prevent accidental changes to the internal state of the map.